[2023-01-28 00:50:41,030] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2023-01-28 00:50:41,058] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2023-01-28 00:50:41,061] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 00:50:41,063] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-01-28 00:50:41,065] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 00:50:41,093] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-02-01 00:00:00+00:00
[2023-01-28 00:50:41,111] {standard_task_runner.py:52} INFO - Started process 11008 to run task
[2023-01-28 00:50:41,119] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_fhv_dag', 'local_to_gcs_task', 'scheduled__2019-02-01T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_fhv_dag.py', '--cfg-path', '/tmp/tmpvaih3fit', '--error-file', '/tmp/tmpsa3gqapi']
[2023-01-28 00:50:41,121] {standard_task_runner.py:77} INFO - Job 31: Subtask local_to_gcs_task
[2023-01-28 00:50:41,197] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [running]> on host adbf820de9e6
[2023-01-28 00:50:41,253] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-01-28 00:50:41,282] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_fhv_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-02-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-01T00:00:00+00:00
[2023-01-28 00:51:09,128] {python.py:175} INFO - Done. Returned value was: None
[2023-01-28 00:51:09,143] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_fhv_dag, task_id=local_to_gcs_task, execution_date=20190201T000000, start_date=20230128T005041, end_date=20230128T005109
[2023-01-28 00:51:09,182] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-01-28 00:51:09,213] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-01-28 00:51:09,232] {dagrun.py:545} INFO - Marking run <DagRun data_ingestion_fhv_dag @ 2019-02-01 00:00:00+00:00: scheduled__2019-02-01T00:00:00+00:00, externally triggered: False> successful
[2023-01-28 00:51:09,233] {dagrun.py:605} INFO - DagRun Finished: dag_id=data_ingestion_fhv_dag, execution_date=2019-02-01 00:00:00+00:00, run_id=scheduled__2019-02-01T00:00:00+00:00, run_start_date=2023-01-28 00:50:30.279163+00:00, run_end_date=2023-01-28 00:51:09.233709+00:00, run_duration=38.954546, state=success, external_trigger=False, run_type=scheduled, data_interval_start=2019-02-01 00:00:00+00:00, data_interval_end=2019-03-01 00:00:00+00:00, dag_hash=9c6f3f2a14e8ddea071f2a17d3a92c71
[2023-01-28 00:52:50,281] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2023-01-28 00:52:50,296] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2023-01-28 00:52:50,297] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 00:52:50,298] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-01-28 00:52:50,299] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 00:52:50,315] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-02-01 00:00:00+00:00
[2023-01-28 00:52:50,322] {standard_task_runner.py:52} INFO - Started process 303 to run task
[2023-01-28 00:52:50,327] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_fhv_dag', 'local_to_gcs_task', 'scheduled__2019-02-01T00:00:00+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_fhv_dag.py', '--cfg-path', '/tmp/tmpuaqe6yj_', '--error-file', '/tmp/tmptcls81d1']
[2023-01-28 00:52:50,330] {standard_task_runner.py:77} INFO - Job 64: Subtask local_to_gcs_task
[2023-01-28 00:52:50,397] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [running]> on host adbf820de9e6
[2023-01-28 00:52:50,450] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-01-28 00:52:50,477] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_fhv_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-02-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-01T00:00:00+00:00
[2023-01-28 00:54:01,950] {python.py:175} INFO - Done. Returned value was: None
[2023-01-28 00:54:01,965] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_fhv_dag, task_id=local_to_gcs_task, execution_date=20190201T000000, start_date=20230128T005250, end_date=20230128T005401
[2023-01-28 00:54:02,011] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-01-28 00:54:02,042] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-01-28 00:58:06,079] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2023-01-28 00:58:06,091] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2023-01-28 00:58:06,092] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 00:58:06,093] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-01-28 00:58:06,093] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 00:58:06,105] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-02-01 00:00:00+00:00
[2023-01-28 00:58:06,111] {standard_task_runner.py:52} INFO - Started process 978 to run task
[2023-01-28 00:58:06,114] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_fhv_dag', 'local_to_gcs_task', 'scheduled__2019-02-01T00:00:00+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_fhv_dag.py', '--cfg-path', '/tmp/tmpohi37xzg', '--error-file', '/tmp/tmp76wn_j9w']
[2023-01-28 00:58:06,116] {standard_task_runner.py:77} INFO - Job 75: Subtask local_to_gcs_task
[2023-01-28 00:58:06,188] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [running]> on host adbf820de9e6
[2023-01-28 00:58:06,235] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-01-28 00:58:06,255] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_fhv_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-02-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-01T00:00:00+00:00
[2023-01-28 00:58:34,595] {python.py:175} INFO - Done. Returned value was: None
[2023-01-28 00:58:34,608] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_fhv_dag, task_id=local_to_gcs_task, execution_date=20190201T000000, start_date=20230128T005806, end_date=20230128T005834
[2023-01-28 00:58:34,659] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-01-28 00:58:34,698] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-01-28 03:33:43,281] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2023-01-28 03:33:43,293] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2023-01-28 03:33:43,295] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 03:33:43,295] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-01-28 03:33:43,296] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 03:33:43,308] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-02-01 00:00:00+00:00
[2023-01-28 03:33:43,314] {standard_task_runner.py:52} INFO - Started process 19001 to run task
[2023-01-28 03:33:43,318] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_fhv_dag', 'local_to_gcs_task', 'scheduled__2019-02-01T00:00:00+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_fhv_dag.py', '--cfg-path', '/tmp/tmp490g8_3i', '--error-file', '/tmp/tmpxjlpexkn']
[2023-01-28 03:33:43,320] {standard_task_runner.py:77} INFO - Job 113: Subtask local_to_gcs_task
[2023-01-28 03:33:43,387] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [running]> on host adbf820de9e6
[2023-01-28 03:33:43,439] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-01-28 03:33:43,460] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_fhv_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-02-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-01T00:00:00+00:00
[2023-01-28 03:33:54,091] {python.py:175} INFO - Done. Returned value was: None
[2023-01-28 03:33:54,107] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_fhv_dag, task_id=local_to_gcs_task, execution_date=20190201T000000, start_date=20230128T033343, end_date=20230128T033354
[2023-01-28 03:33:54,172] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-01-28 03:33:54,215] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-01-28 03:38:19,076] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2023-01-28 03:38:19,097] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2023-01-28 03:38:19,098] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 03:38:19,099] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-01-28 03:38:19,100] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 03:38:19,113] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-02-01 00:00:00+00:00
[2023-01-28 03:38:19,120] {standard_task_runner.py:52} INFO - Started process 19880 to run task
[2023-01-28 03:38:19,126] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_fhv_dag', 'local_to_gcs_task', 'scheduled__2019-02-01T00:00:00+00:00', '--job-id', '152', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_fhv_dag.py', '--cfg-path', '/tmp/tmpks7ir1qq', '--error-file', '/tmp/tmppb62y0a_']
[2023-01-28 03:38:19,128] {standard_task_runner.py:77} INFO - Job 152: Subtask local_to_gcs_task
[2023-01-28 03:38:19,193] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [running]> on host adbf820de9e6
[2023-01-28 03:38:19,238] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-01-28 03:38:19,264] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_fhv_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-02-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-01T00:00:00+00:00
[2023-01-28 03:38:49,474] {python.py:175} INFO - Done. Returned value was: None
[2023-01-28 03:38:49,489] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_fhv_dag, task_id=local_to_gcs_task, execution_date=20190201T000000, start_date=20230128T033819, end_date=20230128T033849
[2023-01-28 03:38:49,515] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-01-28 03:38:49,545] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-01-28 03:40:34,856] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2023-01-28 03:40:34,872] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2023-01-28 03:40:34,873] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 03:40:34,874] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-01-28 03:40:34,875] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-01-28 03:40:34,886] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-02-01 00:00:00+00:00
[2023-01-28 03:40:34,893] {standard_task_runner.py:52} INFO - Started process 20193 to run task
[2023-01-28 03:40:34,896] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_fhv_dag', 'local_to_gcs_task', 'scheduled__2019-02-01T00:00:00+00:00', '--job-id', '159', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_fhv_dag.py', '--cfg-path', '/tmp/tmp2mop2fws', '--error-file', '/tmp/tmp_ibpchas']
[2023-01-28 03:40:34,898] {standard_task_runner.py:77} INFO - Job 159: Subtask local_to_gcs_task
[2023-01-28 03:40:34,966] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_fhv_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [running]> on host adbf820de9e6
[2023-01-28 03:40:35,015] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-01-28 03:40:35,037] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_fhv_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-02-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-01T00:00:00+00:00
[2023-01-28 03:41:00,610] {python.py:175} INFO - Done. Returned value was: None
[2023-01-28 03:41:00,630] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_fhv_dag, task_id=local_to_gcs_task, execution_date=20190201T000000, start_date=20230128T034034, end_date=20230128T034100
[2023-01-28 03:41:00,663] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-01-28 03:41:00,697] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
